### DISCUSSION TITLE
Connection to Azure DataBricks Communication Link Failure - HTTP Response Code 401: Unknown

### DISCUSSION THREAD

**Discussion by meachcarnahan** [dbeaver/dbeaver#20339] [Q&A]
Hello! 

I am trying to make a connection from DBeaver Community (Version 23.1) to Azure Databricks but when I try to test my connection or connect to DataBricks I get the following error: 

[Databricks][DatabricksJDBCDriver](500593) Communication link failure. Failed to connect to server. Reason: HTTP Response code: 401, Error message: Unknown.
  HTTP Response code: 401, Error message: Unknown
  HTTP Response code: 401, Error message: Unknown

This is the connection string

jdbc:spark://adb-3615756102155657.17.azuredatabricks.net:443/default;transportMode=http;ssl=1;httpPath=sql/protocolv1/o/3615756102155657/0111-001654-iluk6w4e;AuthMech=3;UID=token;PWD=*************************************

username: doman\username
password: *********

In the Driver Manager I have downloaded the Databricks jar file DatabricksJDBC42.jar and mapped it in the libraries as described by the DataBricks documentation. 

Any help would be greatly appreciated!

Cheers,
Meachel

**Comment by vibhajani** [comment]
@meachcarnahan did you able to resolve it? I am facing similar error and followed all various options.
